elf/elf.go

```diff
@@ -442,6 +451,18 @@ func (b *Module) relocate(data []byte, rdata []byte) error {
 type SectionParams struct {
 	PerfRingBufferPageCount   int
 	SkipPerfMapInitialization bool
+	CustomConfig              []PinCustomConfig
```

Comment:
`Custom Config` should be named differently I think, e.g. `PinConfig`. It's a very generic name while the field is solely for pinning.

Reply:
Also, I'm not sure if we should have / need a slice of `PinCustomConfig`. For example, what if we add more fields later and those have conflicts?